SobokSobok/SobokSobok/Data/Model/SignUpDataModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  SignUpDataModel.swift
+//  SobokSobok
+//
+//  Created by 김선오 on 2022/01/18.
+//
+
+import UIKit
+
+struct SignUpDataModel {
+    var email: String?
+    var password: String?
+    var name: String?
+}
+
+extension SignUpDataModel {
+    static var shared: SignUpDataModel = SignUpDataModel(email: "", password: "", name: "")
+}
```

Comment:
```suggestion

class SignUpUser {
static let shard = SignUpUser()
var email: String?
var password: String?
var nickname: String?
```

name -> nickname이 더 괜찮을 거 같아용

Reply:
원래 nickname 으로 했었는데 서버통신할때 request body 이름이 name이여서 수정했습니다ㅠㅠ